client/my-sites/domains/domain-management/list/list-header.jsx

```diff
@@ -27,6 +28,24 @@ class ListHeader extends React.PureComponent {
 		const { headerClasses, translate } = this.props;
 
 		const listHeaderClasses = classNames( 'list-header', headerClasses );
+		const emailPopoverText = isEnabled( 'titan/phase-2' )
+			? translate(
+					'You can receive email using your custom domain by purchasing Email or %(googleMailService)s, or by ' +
```

Comment:
> You can receive email using your custom domain by purchasing Email ...

This is a good example of why I don't think `Email` is a good name for this product. It's going to be confusing for users and translators.

Reply:
I agree that it may be complicated in some ways, but it is also simple in other ways. We should be able to complicate the name if we need to!